CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.2] - 2021-05-07
+### Changes
+- Set up publishing to Maven Central (thanks @DeKaN). ([#124](https://github.com/moxy-community/Moxy/issues/124))
+- Update AGP to 4.1.3
```

Comment:
I don't think we need to mention it in changelog - why would one bother what version of AGP was used to build this version?

Reply:
Fair point, removed it.